### ISSUE TITLE
Merging grid cells for bigger rooms

### ISSUE THREAD

**Issue by darkgnostic** [audinowho/RogueElements#8]
Nice work with library!

I am curious is there a possibility to merge two neighbor cells, and build one bigger room out of it?

**Comment by audinowho** [comment]
Yes, a larger grid room spanning multiple grid tiles can be created.  GridPlan.AddRoom contains an overload that takes a Rect object instead of a Loc.  You can perform a "merge" be deleting the rooms already in those cells and then adding a new one that encompasses all rooms you deleted (and nothing more).  It's up to the user to choose which room algorithm gets to drive the resulting large room.
